### PR TITLE
PP-11327 remove username from User class

### PR DESCRIPTION
**Pull request by SandorArpa**
## WHAT

We no longer have username as such stored in adminusers and we are not going to include it in any responses that contain user info.

- remove username from User.class
- refactor code accordingly
- refactor views accordingly
- refactor tests accordingly

### CODE REVIEW

File: test/fixtures/invite.fixtures.js [hunk 1]
```diff
@@ -49,12 +49,6 @@ module.exports = {
     return opts.map(buildInviteWithDefaults)
   },
 
-  notPermittedInviteResponse: (userName, serviceId) => {
```
Comment (alexbishop1) [inline test/fixtures/invite.fixtures.js:52]: I assume this was already completely unused.

File: test/integration/service-users.ft.test.js [hunk 2]
```diff
@@ -29,9 +29,9 @@ chai.use(chaiAsPromised)
 
 describe('service users resource', () => {
   let EXTERNAL_ID_LOGGED_IN = '7d19aff33f8948deb97ed16b2912dcd3'
-  const USERNAME_LOGGED_IN = 'existing-user@example.com'
+  const USEREMAIL_LOGGED_IN = 'existing-user@example.com'
```
Comment (alexbishop1) [inline test/integration/service-users.ft.test.js:32]: `USER_EMAIL_LOGGED_IN`, I think.

File: test/integration/service-users.ft.test.js [hunk 3]
```diff
@@ -29,9 +29,9 @@ chai.use(chaiAsPromised)
...
   const EXTERNAL_ID_OTHER_USER = '393266e872594f1593558549caad95ec'
-  const USERNAME_OTHER_USER = 'other-user@example.com'
+  const USEREMAIL_OTHER_USER = 'other-user@example.com'
```
Comment (alexbishop1) [inline test/integration/service-users.ft.test.js:34]: `USER_EMAIL_OTHER_USER`, I think.

**Review by alexbishop1 — CHANGES_REQUESTED** [review]
I like a PR that reduces the number of lines of code. Just a couple of constant names need tweaking.

File: test/fixtures/invite.fixtures.js (hunk 1 above)
Comment (SandorArpa) [reply test/fixtures/invite.fixtures.js:52]: Yes

File: test/integration/service-users.ft.test.js (hunk 2 above)
Comment (SandorArpa) [reply test/integration/service-users.ft.test.js]: 👍

File: test/integration/service-users.ft.test.js [hunk 4]
```diff
@@ -29,9 +29,9 @@ chai.use(chaiAsPromised)
 
 describe('service users resource', () => {
   let EXTERNAL_ID_LOGGED_IN = '7d19aff33f8948deb97ed16b2912dcd3'
-  const USERNAME_LOGGED_IN = 'existing-user@example.com'
+  const USERE_MAIL_LOGGED_IN = 'existing-user@example.com'
```
Comment (alexbishop1) [inline test/integration/service-users.ft.test.js:32]: 🤨

**Review by alexbishop1 — CHANGES_REQUESTED** [review]

**Review by alexbishop1 — APPROVED** [review]

**Pull request merged**